src/style.css

```diff
@@ -2,2 +1,97 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+h1 {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+}
+
+header {
+  width: 100%;
+  background-color: #fff;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.logo img {
+  height: 40px;
+}
+
+nav ul li a {
```

Comment:
To ensure that the navigation items are displayed in uppercase, consider adding 'text-transform: uppercase;' to this selector. This will make your HTML more semantic by not requiring the text to be hardcoded in uppercase.